MetaMorpheus/Test/PostSearchAnalysisTaskTests.cs

```diff
@@ -101,6 +101,16 @@ public static void AllResultsAndResultsTxtContainsCorrectValues_PepQValue_Bottom
             Assert.AreEqual("TaGe_SA_A549_3_snip_2 - Target protein groups within 1 % FDR: 140", results[15]);
         }
 
+        [Test]
+        public static void TestPeptideQuantificationResults()
```

Comment:
This test seems subject to break in the future due to the hard coded number. Could you instead read in the peptides file, count the results at threshold, then use that in your assert?